server/events/yaml/valid/policies.go

```diff
@@ -0,0 +1,20 @@
+package valid
+
+// Policies defines version of policy checker binary(conftest) and a list of
+// PolicySet objects. Policies struct is used by PolicyCheck workflow to build
+// context to enforce policies.
+type Policies struct {
+	Version    string
+	PolicySets []PolicySet
+}
+
+type PolicySet struct {
+	Source PolicySetSource
+	Name   string
+	Owners []string
+}
+
+type PolicySetSource struct {
+	Type string
+	Path string
+}
```

Comment:
is this different than what already exists in models.policy? We should dedup it.

Reply:
I haven't gotten to models.PolicySets yet. But looks like project config is not being passed to ProjectCommandContext object so we need something there to represent the policies defined in the yaml file. And current code structure doesn't reuse models in the yaml package or vise versa. It is a bit of duplication but there is a potential that models.PolicySets will have different structure or will have additional functionality that will warrant the separate struct.

Reply:
Damn thats quite unfortunate. let's keep the naming consistent then between the two?

Reply:
Looks I can just reuse `valid.PolicySets` instead of using `models.PolicySets`. That should remove unnecessary struct from models